CheckboxItem/CheckboxItem.module.less

```diff
@@ -10,6 +10,8 @@
 		padding-left: @agate-checkboxitem-padding-left;
 		padding-right: @agate-checkboxitem-padding-right;
 		border-color: @agate-checkboxitem-border-color;
+		border-bottom-color: @agate-checkboxitem-border-bottom-color;
```

Comment:
I might break the gallium CheckboxItem GUI.
The border-bottom-color in Gallium will be not `transparent` but `inherit`.

color--gallium-day.less
```
@agate-checkboxitem-border-color: transparent;
```